source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm

```diff
@@ -385,8 +485,24 @@ - (NSMutableDictionary *)getTextMap
     return _textMap;
 }
 
+- (NSMutableDictionary *)getActionsMap
+{
+    return _actionsMap;
+}
+
 - (ACOAdaptiveCard *)card
 {
     return _adaptiveCard;
 }
+
+- (UIImage*)scaleImage:(UIImage *)image toSize:(CGSize)newSize {
+    //UIGraphicsBeginImageContext(newSize);
+    // In next line, pass 0.0 to use the current device's pixel scaling factor (and thus account for Retina resolution).
+    // Pass 1.0 to force exact pixel size.
+    UIGraphicsBeginImageContextWithOptions(newSize, NO, 0.0);
```

Comment:
this is expensive operation. prefer NSLayoutConstraint and its member function constraintWithItem